ansible/roles/bootstrap/tasks/partials/aix.yml

```diff
@@ -0,0 +1,49 @@
+---
+
+#
+# Setup the filesystem to the correct sizes on AIX
+#
+
+- name: Set size of / to 1G
+  aix_filesystem:
```

Comment:
Is this part of the standard ansible install? Do we need to update instructions to use this?

Reply:
it should be part of standard past a certain version 2.8+ im on 2.8.5 and didnt need to install anything extra